JIRA-OSDOCS3384: Updated limited support status in service definition (#43911)
openshift/openshift-docs: modules/life-cycle-limited-support.adoc

[id="rosa-limited-support_{context}"]
= Limited support status

> When a cluster transitions to a _Limited Support_ status, it means that the SLA is no longer applicable and credits requested against the SLA are denied. It does not mean that you no longer have product support. The cluster can return to a fully-supported status if the violating factors are remediated.

Author: Made the edit.

> When a cluster transitions to a _Limited Support_ status, it means that the SLA is no longer applicable and credits requested against the SLA are denied. It does not mean that you no longer have product support. The cluster can return to a fully-supported status if the violating factors are remediated.

Author: Went with the first suggestion. Thank you!

A cluster transitions to a _Limited Support_ status in the following scenarios:

> * If you do not upgrade a cluster to a supported version before the end-of-life date

Author: Excellent idea! I tried this out and the output format is way better!! Thank you, @jeana-redhat

+

> Red Hat does not make any runtime or SLA guarantees for versions after their end-of-life date. To receive continued support, upgrade the cluster to a supported version within the 9 month period. If you do not upgrade the cluster within the 9 month period, the cluster transitions to a _Limited Support_ status until you upgrade it to a supported version.

Author: Made the edit.

+
Red Hat provides commercially reasonable support to upgrade from an unsupported version to a supported version. However, if a supported upgrade path is no longer available, you might have to create a new cluster and migrate your workloads.

> * If you remove or replace any native {product-title} components or any other component that is installed and managed by Red Hat

Reviewer: ISG says to avoid the use of the word "native" unless the term is part of the existing product terminology, for example, “cloud native”. The suggested words they have for alternatives ("local", "basic", "original", or "default") don't make sense here (imo), but the way this is written, it sounds like native components are a subset of components that are installed and managed by Red Hat. Therefore, would this work?
```suggestion
* If you remove or replace any components that are installed and managed by Red Hat
```

Author: Retained the existing term, to specifically call out the native ROSA components as a special subset of the components RH manages per discussion with Jeana. Thank you.

+

> If you used the cluster administrator rights, Red Hat is not responsible for any of your or your authorized users’ actions, including those that affect the infrastructure services, service availability, or data loss. If Red Hat detects any such actions, the cluster transitions to a _Limited Support_ status. Red Hat notifies you of such and you should either revert the action or create a support case to remediate the issue.

Reviewer 2: `your or your` I think should be `yours or your`

Author: No, it doesn't play well grammatically. Retained the existing wording.

> If you used the cluster administrator rights, Red Hat is not responsible for any of your or your authorized users’ actions, including those that affect the infrastructure services, service availability, or data loss. If Red Hat detects any such actions, the cluster transitions to a _Limited Support_ status. Red Hat notifies you of such and you should either revert the action or create a support case to remediate the issue.

Reviewer: ```suggestion
If you used the cluster administrator rights, Red Hat is not responsible for any of your or your authorized users’ actions, including those that affect infrastructure services, service availability, or data loss. If Red Hat detects any such actions, the cluster transitions to a _Limited Support_ status. Red Hat notifies you of the status change and you should either revert the action or create a support case to remediate the issue.
```

Author: Made the change.
